src/Services/Data/DefaultEmoteRegexMatchWriter.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace BrackeysBot
+{
+    public class DefaultEmoteRegexMatchWriter
+    {
+        /// <summary>
+        /// Creates a file that contains the c# dictionary definition for the discord default emote map with information about 
+        /// how many matches for each emote are found when checked against the regex pattern of all default emotes. This is to 
+        /// counteract false positives by using (foundEmoteMatchesForEmoteType / regexMatchCountForSingleEmote)
+        /// </summary>
+        /// <param name="filePath">path at which the resulting file is stored</param>
+        public static void WriteEmoteValueSet(string filePath)
+        {
+            string[] emoteValuesRegexEscaped = DiscordDefaultEmoteData.EmoteMap.Select(x => Regex.Escape(x.Value)).ToArray();
+
+            int GetMatchCountAgainstAllEntries(string msg)
```

Comment:
Might consider just switching to a LINQ solution instead of a local method for better readability.

Reply:
Sure. I don't think it is less readable, but that can be done of course.